README.md

```diff
@@ -12,97 +12,59 @@ yarn add [-D] webpacker
 
 # Usage
 ## Commands
-For now there are 2 commands you can use.
-- `serve` => Webpacker will run in development mode and serve your project on port `4000`.
-- `build` => Webpacker will run in production mode and build + minify/uglify your project and output it to a folder.
+For now there are 3 commands you can use.
+- `serve` => Webpacker will serve your project for development purposes.
+- `build` => Webpacker will build your project and output it to a folder.
+- `stats` => Webpacker will build your project and output it to a folder and it will open a browser window with information about your bundle sizes.
 
 ## Dotfile
-To provide configuration to webpacker, you'll need to add a `.webpacker.json` file in the root of your project. The data inside the dotfile will also be available in your project's source code, which will be explained [here](#environmental-config).
+To provide configuration to webpacker, you'll need to add a `.webpacker.js` file in the root of your project. The location of the config file can be changed by passing `--config path/to/my/file.js` when running webpacker.
 
-- `index` => The location of your index.ejs file (always has to be .ejs to be able to provide dynamic options).
-- `entry` => The location of your index.js file, the entry point to your app.
-- `app` => The location of the src folder of your app.
-- `config` => The location of the environment config folder of your app. The environment name you pass to webpacker will be used to grab the file (so `production` would tell webpacker to use `<config>/production.js`).
-- `output` => The output folder of the build process (only required when running the `build` command).
-- `presets` => Array of strings to specify the presets that should be used. You can provide relative file paths to custom presets (more about that [here](#custom-presets)).
-- `scss` => The folder location of your scss files. This will make sure that webpacker can grab the `variables.scss` file when using the `scss` preset.
+The `.webpacker.js` file needs to export an object with at least the `preset` key.
 
-### Example
-`.webpacker.json`
+### Preset
+The preset defines which loaders and plugins will be used. If the loader or plugin you need is not available, please submit an issue or a PR.
```

Comment:
Should we move presets to different repositories so we decouple `webpacker` from the specific presets?

Reply:
Presets are defined by the user of webpacker, we just provide the loaders/plugins. I think it makes sense to keep them in the same repo, since we already have a kind of opinionated way of using them

Reply:
Maybe once we have a lot of them we could split them somehow

Reply:
We could also transform this repo into  a monorepo containing multiple packages.